kivy/uix/scrollview.py

```diff
@@ -764,7 +764,8 @@ def on_touch_move(self, touch):
         if touch.grab_current is not self:
             return True
 
-        if not any(key.startswith('sv.') for key in touch.ud):
+        if not any(isinstance(key, str) and key.startswith('sv.')
+                   for key in touch.ud):
```

Comment:
In python, `not` binds before `and`. So you need parentheses around the statement `if not (x and y)`.